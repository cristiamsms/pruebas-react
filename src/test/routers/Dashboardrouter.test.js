const { shallow } = require("enzyme")
const { Dashboardrouter } = require("../../routers/Dashboardrouter")
import React from 'react';
describe('pruebas en el dashboard', () => {
    test('prueba 1', () => {
        const wrapper =shallow(<Dashboardrouter/>)
        expect(wrapper).toMatchSnapshot();
    })
    
    
})
