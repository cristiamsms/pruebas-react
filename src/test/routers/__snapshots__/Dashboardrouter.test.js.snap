// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en el dashboard prueba 1 1`] = `
<Fragment>
  <Navbar />
  <div
    className="container mt-4 ml-4"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/marvel"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/hero/:heroeId"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/dc"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/Search"
      />
      <Redirect
        to="/marvel"
      />
    </Switch>
  </div>
</Fragment>
`;
